frontend/src/components/Header.vue

```diff
@@ -12,129 +12,36 @@
+        <v-btn class="title hidden-sm-and-down" color="text" flat>Natural Resources Get Token</v-btn>
       </v-toolbar-title>
-
-      <v-spacer></v-spacer>
-
-      <!-- <v-menu bottom left>
-        <template v-slot:activator="{ on }">
-          <v-btn icon v-on="on">
-            <v-icon x-large>menu</v-icon>
-          </v-btn>
-        </template>
-        <v-layout align-center justify-center row fill-height>
-          <v-flex xs6>
-            <v-list class="header-menu">
-              <v-list-tile v-for="(item, key) in menuSecondary" :key="'secondary-menu-'+key">
-                <v-btn v-if="item.link" flat :to="item.link" class="navbar-link lvl2-link">
-                  <v-icon v-if="item.icon">{{item.icon}}</v-icon>
-                  {{ item.title }}
-                </v-btn>
-                <v-btn v-else flat :href="item.href" class="navbar-link lvl2-link">
-                  <v-icon v-if="item.icon">{{item.icon}}</v-icon>
-                  {{ item.title }}
-                </v-btn>
-              </v-list-tile>
-            </v-list>
-          </v-flex>
-          <v-flex xs6>
-            <v-list class="header-menu-secondary">
-              <v-list-tile v-for="(item, key) in menuTertiary" :key="'tertiary-menu-'+key">
-                <v-btn v-if="item.link" flat :to="item.link" class="navbar-link lvl2-link">
-                  <v-icon v-if="item.icon">{{item.icon}}</v-icon>
-                  {{ item.title }}
-                </v-btn>
-                <v-btn v-else flat :href="item.href" class="navbar-link lvl2-link">
-                  <v-icon v-if="item.icon">{{item.icon}}</v-icon>
-                  {{ item.title }}
-                </v-btn>
-              </v-list-tile>
-            </v-list>
-          </v-flex>
-        </v-layout>
-      </v-menu> -->
     </v-toolbar>
   </header>
 </template>
 
 <script>
-  export default {
-    data: () => ({
-      logInUrl: '/api/login',
-      searchText: '',
-      showSearch: false,
-      classicUrl: '',
-      menuSecondary: [
-        {
-          title: 'What is NR Get Token?',
-          href: 'https://developer.gov.bc.ca/'
-        },
-        {
-          title: 'Placeholder',
-          href: 'https://developer.gov.bc.ca/'
-        },
-        {
-          title: 'Developers',
-          href: 'https://www.bcdevexchange.org/'
-        },
-        {
-          title: 'Contact',
-          href: 'https://developer.gov.bc.ca/'
-        }
-      ]
-    }),
-    computed: {
-      menuTertiary: () => ([
-        {
-          title: 'Datasets',
-          link: '/datasets'
-        },
-        {
-          title: 'Organizations',
-          link: '/organization'
-        },
-        {
-          title: 'Groups',
-          link: '/groups'
-        },
-        {
-          title: 'About',
-          link: '/about'
-        }
-      ])
-    },
-  };
+export default {};
```

Comment:
I think we might be able to even omit this line and have it just be empty if we're effectively exporting nothing.

Reply:
Yup, works without it.